cranelift/codegen/src/ir/function.rs

```diff
@@ -3,6 +3,8 @@
 //! The `Function` struct defined in this module owns all of its basic blocks and
 //! instructions.
 
+#![allow(unused_imports)]
```

Comment:
This may be an artifact of development, but we should avoid adding warning overrides like this in the main tree unless there's a practical reason we can't just fix the warning.